app/controllers/applications/supplementals_controller.rb

```diff
@@ -3,9 +3,11 @@ module Applications
   class SupplementalsController < ApplicationController
     before_action :authenticate_user!
     def index
-      @application = application_service.application(params[:application_id])
+      includes = %w[preferences proof_files household_members flagged_applications lease]
```

Comment:
what's the purpose behind this?

Reply:
`application_service.application` is querying multiple tables, it's just not one query. I added the `include` option so you can specified what attributes to pull, so it does not have to make query for a nested attribute if it's not required. 

I had to add the `lease` attribute to  the application query since it's being used here (suppl app). However it's not being used in application new and edit and they use the same query. So I added the include to avoid running the query to pull the lease in application new and edit